src/lang.js

```diff
@@ -20,3 +54,4 @@
   };
   window.getString = getString;
 }(window));
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the end of file. Max of 0 allowed.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)